### ISSUE TITLE
undefined method `next_element' for nil:NilClass when render template

### ISSUE THREAD

**Issue by Xorg1984** [senny/sablon#21]
when i try to render some templates i get an error - undefined method `next_element' for nil:NilClass
An error occurs in file mail_merge.rb line 36

**Comment by senny** [comment]
@Xorg1984 thank you for reporting. Please go into more details about the issue. With the information given there is no way I can debug your problem. A good start would be to upload your template (docx) and the sablon script (ruby) somewhere. This way I can take a look and hopefully reproduce the issue you are facing.

**Comment by Xorg1984** [comment]
To reproduce this issue try run this test script

```ruby
require "sablon"

template = Sablon.template('ddu_6_test.docx')
context = {}
template.render_to_string(context)
```
Template file [ddu_6_test.docx](https://dl.dropboxusercontent.com/u/545979/ddu_6_test.docx)

When i run this script i get an error:
/home/dm/.rvm/gems/ruby-2.2.1/gems/sablon-0.0.14/lib/sablon/parser/mail_merge.rb:36:in `pattern_node': undefined method `next_element' for nil:NilClass (NoMethodError)

**Comment by senny** [comment]
@Xorg1984 Thanks for the additional information.

By inspecting your document I discovered that it did contain an incomplete merge field. I can't tell you how that ended up in your `document.xml` but it makes sense for sablon to not crash and ignore invalid fields. I pushed a patch and released a new version `0.0.15` to rubygems.org.

Let me know if the new version works for you.

**Comment by Xorg1984** [comment]
Thank you! after update sablon issue not repeated
